client/lib/wpcom-undocumented/lib/site.js

```diff
@@ -183,6 +184,20 @@ UndocumentedSite.prototype.setOption = function( query, callback ) {
 	);
 }
 
+UndocumentedSite.prototype.postCounts = function( options, callback ) {
+	var type,
+		query = merge( {
+			type: 'post',
+			apiNamespace: 'wp', // This currently has no bearing on the route, wpcom-xhr-request needs to be updated to remove the namespace whitelist
+			locale: i18n.getLocaleSlug()
```

Comment:
Rather than do this manually, you can make use of [`Undocumented.prototype._sendRequestWithLocale`](https://github.com/Automattic/wp-calypso/blob/58d03103ee4139d727e19aca74ecf2178bf5365b/client/lib/wpcom-undocumented/lib/undocumented.js#L358-L371) or (and my preference), [`wpcom.withLocale()`](https://github.com/Automattic/wp-calypso/tree/master/client/lib/wp/localization)